shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -116,39 +115,39 @@ - (void)dealloc {
 - (NSMethodSignature*)methodSignatureForSelector:(SEL)sel {
   NSMethodSignature* result = [super methodSignatureForSelector:sel];
   if (!result) {
-    result = [_nativeSwitch methodSignatureForSelector:sel];
+    result = [self.nativeSwitch methodSignatureForSelector:sel];
   }
   return result;
 }
 
 - (void)forwardInvocation:(NSInvocation*)anInvocation {
-  [anInvocation setTarget:_nativeSwitch];
+  anInvocation.target = self.nativeSwitch;
   [anInvocation invoke];
 }
 
 - (NSString*)accessibilityValue {
-  if ([self node].HasFlag(flutter::SemanticsFlags::kIsToggled) ||
-      [self node].HasFlag(flutter::SemanticsFlags::kIsChecked)) {
-    _nativeSwitch.on = YES;
+  if (self.node.HasFlag(flutter::SemanticsFlags::kIsToggled) ||
+      self.node.HasFlag(flutter::SemanticsFlags::kIsChecked)) {
+    self.nativeSwitch.on = YES;
   } else {
-    _nativeSwitch.on = NO;
+    self.nativeSwitch.on = NO;
   }
```

Comment:
Consider changing to

```objc
self.nativeSwitch.on = self.node.HasFlag(flutter::SemanticsFlags::kIsToggled) ||
      self.node.HasFlag(flutter::SemanticsFlags::kIsChecked)
```

to avoid the duplicate assignment statement.